lib/git-shell-out-strategy.js

```diff
@@ -42,6 +43,9 @@ export class LargeRepoError extends Error {
   }
 }
 
+// ignored for the purposes of usage metrics tracking because they're noisy
+const IGNORED_GIT_COMMANDS = ['cat-file', 'config', 'diff', 'for-each-ref', 'log', 'rev-parse', 'status'];
```

Comment:
Good question.  My thinking on this was that

1) I don't really know which commands are important for understanding user behavior.  So logging all of them except really obviously noisy ones avoids having to be more specific about what we'd like to learn from this.  Which could be good or bad depending on how you look at it.

2) Using an "exclude" list rather than an "include" list means that we don't have to worry about forgetting to update the list when we add support for new git commands. (Although obviously if we add new noisy commands, we'd still need to update.)